app/src/main/java/com/framgia/englishconversation/screen/videoDetail/VideoDetailViewModel.java

```diff
@@ -34,13 +37,38 @@
     private Navigator mNavigator;
     private FragmentManager mManager;
     private CommentFragment mFragment;
-
-    VideoDetailViewModel(Context context, FragmentManager manager, TimelineModel timelineModel) {
+    private UserModel mTimelineUser;
+    private OnTimelineItemTouchListener mTouchListener =
+            new OnTimelineItemTouchListener<TimelineModel>() {
+                @Override
+                public void onItemTimelineClick(TimelineModel item) {
+                    //TODO
+                }
```

Comment:
ERROR: 'method def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.